### ISSUE TITLE
New optional secondary option for:  double-slash-comment-empty-line-before

### ISSUE THREAD

**Issue by dperolio** [kristerkari/stylelint-scss#343]
It would be nice to have an option of `except: ["inside-block"]` or even `ignore: ["inside-block"]` as stylelint's standard [rule-empty-line-before](https://stylelint.io/user-guide/rules/rule-empty-line-before/#except-after-rule-after-single-line-comment-inside-block-and-after-rule-inside-block-first-nested) does. I like to include an empty line before my comments when they are not in blocks, and I like no lines between comments/selectors inside of nested blocks. As it currently stands, I don't think there is any way for me to write code like this: ![image](https://i.imgur.com/dbsxQQ8.png)
Currently, with
```
    "scss/double-slash-comment-empty-line-before": [
      "always",
      {
        "ignore": "between-comments"  
      }
    ],
    "rule-empty-line-before": [
      "always",
      {
        "except": [
          "after-single-line-comment",
          "inside-block"
        ]
      }
    ]
```
I will get this ![image](https://i.imgur.com/SOi1YHf.png)
or with 
```
    "scss/double-slash-comment-empty-line-before": "never",
    "rule-empty-line-before": [
      "always",
      {
        "except": [
          "after-single-line-comment",
          "inside-block"
        ]
      }
    ]
```
I will get this ![image](https://i.imgur.com/EblNpVI.png)
neither of which are optimal. If there's some way around this already, please let me know. Setting `scss/double-slash-comment-empty-line-before` to null seems to be the only way to do as I want, but then it has to be spaced manually, and can't be enforced. I think this would be a good addition. Thanks.

**Comment by kristerkari** [comment]
Thanks for this! Looking at your examples, I guess it makes sense to add the option.

**Comment by srawlins** [comment]
Since stylelint's `rule-empty-line-before` offers an "except" option _and_ an "ignore" option for "inside-block", I think it makes sense to offer both options here:

### `except: ["inside-block"]`

This would _reverse_ the primary option for comments that are inside a block. If the primary option is "always", then empty lines would _never_ be allowed before a comment in a block.

### `ignore: ["inside-block"]`

This would _reverse_ the primary option for comments that are inside a block. Comments inside blocks would be allowed with and without empty lines before.

**Comment by kristerkari** [comment]
This is now fixed in v3.15.0
https://github.com/kristerkari/stylelint-scss/releases/tag/3.15.0

Thanks everyone for helping out!
